app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -214,8 +229,11 @@ function ProjectFramework(props: Props) {
         offset,
     ]);
 
-    const totalFrameworksCount = frameworksResponse?.count ?? 0;
+    const totalFrameworksCount = projectUsersResponse?.analysisFrameworks?.totalCount ?? 0;
+    const AfList: FrameworkMini[] = projectUsersResponse?.analysisFrameworks?.results || [];
 
+    console.log('type one', typeof frameworkList);
+    console.log('type two >> ', projectUsersResponse?.analysisFrameworks?.results);
```

Comment:
Please make sure there are no console logs left before you sent the PR.
My recommendation is doing a self code review as soon as the PR is sent.